thing-url-adapter.js

```diff
@@ -38,6 +38,28 @@ const POLL_INTERVAL = 5 * 1000;
 const WS_INITIAL_BACKOFF = 1000;
 const WS_MAX_BACKOFF = 30 * 1000;
 
+const JWT_AUTH = {}; // Auth tokens to access things with
+// gets filled in loadThingURLAdapter
+
+// This function checks the supplied url against keys
+// in the jwt_auth object.
+function getHeader(url, contentType = false) {
```

Comment:
nit: this should be named `getHeaders()`.

Reply:
Yes it should